Test Failure: PersistentExecutorCompatibilityTest.initializationError NoClassDefFoundError

Test Failure: com.ibm.ws.concurrent.persistent.fat.compat.PersistentExecutorCompatibilityTest.initializationError

```
initializationError:java.lang.RuntimeException: The HttpServlet referenced by the @TestServlet(servlet=PersistentExecCompatibilityTestServlet.class contextRoot='persistentcompattest') annotation imported a class that was not available to the JUnit classpath. Make sure that the missing type is present on the runtime classpath of the FAT  (i.e. somewhere in autoFVT/lib/ or autoFVT/build/lib/ )
at componenttest.annotation.processor.TestServletProcessor.getTestServletMethods(TestServletProcessor.java:104)
at componenttest.annotation.processor.TestServletProcessor.getServletTests(TestServletProcessor.java:67)
at componenttest.custom.junit.runner.FATRunner.getChildren(FATRunner.java:154)
at componenttest.custom.junit.runner.FATRunner.<init>(FATRunner.java:137)
at java.lang.reflect.Constructor.newInstance(Constructor.java:542)
at java.lang.reflect.Constructor.newInstance(Constructor.java:542)
at java.lang.reflect.Constructor.newInstance(Constructor.java:542)
Caused by: java.lang.NoClassDefFoundError: com.ibm.websphere.concurrent.persistent.PersistentExecutor
at java.lang.J9VMInternals.prepareClassImpl(Native Method)
at java.lang.J9VMInternals.prepare(J9VMInternals.java:283)
at java.lang.Class.getMethods(Class.java:1334)
at componenttest.annotation.processor.TestServletProcessor.getTestServletMethods(TestServletProcessor.java:102)
Caused by: java.lang.ClassNotFoundException: com.ibm.websphere.concurrent.persistent.PersistentExecutor
at java.net.URLClassLoader.findClass(URLClassLoader.java:603)
at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:786)
at java.lang.ClassLoader.loadClass(ClassLoader.java:760)
at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:341)
at java.lang.ClassLoader.loadClass(ClassLoader.java:741)
```

This fails when running with Java 7, which the test doesn't need to be running against.

--- a/dev/com.ibm.ws.concurrent.persistent_fat_compat/.classpath
+++ b/dev/com.ibm.ws.concurrent.persistent_fat_compat/.classpath
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
 		<attributes>
 			<attribute name="module" value="true"/>
 		</attributes>

--- a/dev/com.ibm.ws.concurrent.persistent_fat_compat/bnd.bnd
+++ b/dev/com.ibm.ws.concurrent.persistent_fat_compat/bnd.bnd
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2019 IBM Corporation and others.
+# Copyright (c) 2019,2020 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
@@ -10,6 +10,9 @@
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
 
 src: \
 	fat/src,\

